lib/address-derivation-discovery/lib/Cardano/Byron/Codec/Cbor.hs

```diff
@@ -44,39 +44,62 @@ module Cardano.Byron.Codec.Cbor
 import Prelude
 
 import Cardano.Address.Derivation
-    ( XPub, xpubToBytes )
+    ( XPub
+    , xpubToBytes
+    )
```

Comment:
This style has the final `)` on a separate line -- this is the same as `fourmolu`.